### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by Mayitzin** [Mayitzin/ahrs#15] [General]
I just want to thank the people that have used the package. The feedback and your support in all forms has kept me improving it.

I remember when I was a student and struggling to understand many of the AHRS concepts. It felt more a torture when all was about the maths and there wasn't anything practical about it.

Luckily I crossed paths with some people, who helped me to better learn all this.

Now I want to share as much as I can. I want to prevent such burdens to any curious maker, student or scientist. Nothing shall stop you.

I hope this small effort can help you, dear reader, to apply your knowledge and materialize your projects.

Simplicity is the way to go, in my opinion, whereas knowledge should fly and ignite your own ideas.

Thanks again for your comments and support, and let's make the next year a much better one than this. Don't stop making, and learn every day!

Mario.

**Comment by zhanghao0827** [comment]
Thank you, thank you author.
